### ISSUE TITLE
Suggestion: Slider step for changing value

### ISSUE THREAD

**Issue by Bulldog83** [shedaniel/ClothConfig#30]
Can you add the ability to set the step of changing the value for the Sliders?

**Comment by shedaniel** [comment]
it is a limitation on Minecraft vanilla's part. I will see what I can do

**Comment by TelepathicGrunt** [comment]
Yeah this would be super helpful or have it when selecting a slider, you can type in a value. Right now, I have a slider with a range of 0 to 1000 and it's impossible to get certain numbers like 1,2, or 3 since it just jumps between 0 and 4 for me
